src/index.html

```diff
@@ -13,6 +13,42 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="imac-logo"
+      />
+      <div class="card__name-panel">
+        <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      </div>
+      <div class="card__product-code">
+        <p>Product code: 195434</p>
+      </div>
+      <div class="card__review">
+        <div class="stars">
```

Comment:
Add the modifier classes
```suggestion
        <div class="stars stars--4">
```